python/cudf/cudf/pandas/fast_slow_proxy.py

```diff
@@ -937,8 +937,11 @@ def _fast_slow_function_call(
             domain="cudf_pandas",
         ):
             slow_args, slow_kwargs = _slow_arg(args), _slow_arg(kwargs)
-            with disable_module_accelerator():
+            if args[0].__name__ in ("read_pickle", "to_pickle"):
                 result = func(*slow_args, **slow_kwargs)
```

Comment:
I really don't like this one sorry. Since this leaks implementation details of the pandas wrappers into the generic fast slow proxy. Why do you need this? Ideally we would address the root cause so that special-casing like this is not necessary.

Reply:
I think you've managed to avoid this by running them with module accelerator disabled (so we always do fallback to slow)?